e2e/tests/node.go

```diff
@@ -118,9 +118,9 @@ func (p *pocketdBin) runCurlPostCmd(rpcUrl string, service string, data string,
 	dataStr := fmt.Sprintf("%s", data)
 	urlStr := fmt.Sprintf("%s/%s", rpcUrl, service)
 	base := []string{
-		"-v",                                   // verbose output
-		"-sS",                                  // silent with error
-		"POST",                                 // HTTP method
+		"-v",         // verbose output
+		"-sS",        // silent with error
+		"-X", "POST", // HTTP method
```

Comment:
I don't know how this worked before, but `POST` is an argument for `-X` - tests were failing on my machine until I changed that.

Reply:
This is really weird...